satellite/metainfo/metainfo.go

```diff
@@ -325,7 +326,9 @@ func (endpoint *Endpoint) createOrderLimitsForSegment(ctx context.Context, point
 
 		node, err := endpoint.cache.Get(ctx, piece.NodeId)
 		if err != nil {
-			return nil, err
+			endpoint.log.Error("error getting node from overlay cache")
```

Comment:
We should log the actual error too, i.e. something like: 
```
endpoint.log.Error("error getting node from overlay cache", zap.Error(err))
```

Reply:
should we log the nodeID too or no?

Reply:
Yes

Reply:
ok we ended up doing this a level down inside satellite/satellitedb/overlaycache.go